### PR TITLE
MF-571 - Add batch things-channels connection modification

### PR DESCRIPTION
**Pull request by darkodraskovic**
Signed-off-by: Darko Draskovic <darko.draskovic@gmail.com>

### What does this do?

Adds working things-channels connection PUT and DELETE batch request via checkbox list of things and channels.

### Have you included tests for your changes?

No.

### Did you document any new/modified functionality?

No.

### CODE REVIEW

**Review by drasko — APPROVED** [review]
LGTM

File: ui/src/Connection.elm [hunk 1]
```diff
@@ -115,11 +110,19 @@ update msg model token =
...
+checkEntity : String -> List String -> List String
+checkEntity id checkedEntitiesIds =
+    if List.member id checkedEntitiesIds then
+        List.Extra.remove id checkedEntitiesIds
+
+    else
+        id :: checkedEntitiesIds
 
 
 view : Model -> String -> Html Msg
```
Comment (blokovi) [inline ui/src/Connection.elm]: Why do you need token as argument in view? It's not used

**Review by blokovi — CHANGES_REQUESTED** [review]

File: ui/src/Connection.elm (hunk 1 above)
Comment (darkodraskovic) [reply ui/src/Connection.elm]: Removed in the last commit.

**Comment by blokovi** [comment]
LGTM

**Review by blokovi — APPROVED** [review]

**Pull request merged**